packages/@aws-cdk/custom-resources/lib/provider-framework/runtime/consts.ts

```diff
@@ -5,3 +5,4 @@ export const WAITER_STATE_MACHINE_ARN_ENV = 'WAITER_STATE_MACHINE_ARN';
 export const FRAMEWORK_ON_EVENT_HANDLER_NAME = 'onEvent';
 export const FRAMEWORK_IS_COMPLETE_HANDLER_NAME = 'isComplete';
 export const FRAMEWORK_ON_TIMEOUT_HANDLER_NAME = 'onTimeout';
+export const FRAMEWORK_HANDLER_TIMEOUT = 900000; // 15 minutes
```

Comment:
Why is this needed here? Feels like it could be internal to `outbound.ts`? This file is the "protocol" between the construct and the handler.

Reply:
no real reason, except to keep the constants in a single place

Reply:
The file name is misleading. Please move locally.

Reply:
Still waiting for this